ubuntudesign/www.ubuntu.com
# Redirect /usn* to usn.ubuntu.com*

Currently, https://www.ubuntu.com/usn is actually routed to a different application.

We're going to move this page to https://usn.ubuntu.com and disable the existing routing,
at which point requests to `/usn` will fall to this application.

Therefore, I'm adding a redirect for when this happens to send all `/usn` requests
to the correct place.

See [this RT (requires authorization)](https://portal.admin.canonical.com/97233) for more details.

QA

---

Run the site and visit `/usn`, `/usn/` and `/usn/whatever` and check they all redirect to `https://usn.ubuntu.com/xxxx` as expected.
